.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[Bug]"
```

Comment:
Could you add `insert summary` across all PRs?

```suggestion
title: "[Bug] insert summary"
```

Reply:
Will do. Thanks